modules/statics/src/base.ts

```diff
@@ -1533,7 +1533,6 @@ export enum UnderlyingAsset {
   TGBP = 'tgbp',
   TUSDS = 'tusds',
   TGOUSD = 'tgousd',
-  GOUSD = 'gousd',
```

Comment:
why do we need to change the base to eth: gousd?, only name needs to be changed?